src/route-handlers/list-workflows/helpers/get-list-workflow-executions-query.ts

```diff
@@ -1,8 +1,7 @@
 import { type SortOrder } from '@/utils/sort-by';
+import { WORKFLOW_STATUSES } from '@/views/shared/workflow-status-tag/workflow-status-tag.constants';
 import type { WorkflowStatus } from '@/views/shared/workflow-status-tag/workflow-status-tag.types';
```

Comment:
Looks weird that you have both workflow-status-tag.constants and workflow-status-tag.types. I'd converge them in one file.

Reply:
The convention we follow is that .types files are meant to have only Typescript types, and .constants files have only constants.

I can discuss this with Assem once he's back